openfisca_us/parameters/gov/states/md/tax/income/credits/ctc/age_limit.yaml

```
@@ -0,0 +1,12 @@
+description: Maximum age for a child to be eligible for the MD CTC.
+values:
+  2021-01-01:
```

Don't need value nested in date unless you're providing date-specific metadata
```suggestion
  2021-01-01: 17
```